admin.go

```diff
@@ -167,6 +170,27 @@ func handleLoadConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func handleStop(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		http.Error(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
+		return
+	}
+	log.Println("[ADMIN] Initiating shutdown")
+	if err := stopandCleanup(); err != nil {
+		log.Printf("[ADMIN][ERROR] stopping: %v \n", err)
+	}
+	log.Println("[ADMIN] Exiting")
```

Comment:
I feel like this is still needed due to odd behaviour on windows in the console.  After exiting the cursor is not shown in normal place (see below)
![ScreenHunter_190 16-Jul-19 19 55](https://user-images.githubusercontent.com/325502/61321600-c41b4300-a803-11e9-9de5-2ab7625f609c.jpg)

I can now type 'dir' and it will work, but it looks odd.  If I leave out Exiting then its unclear if process is finished.  There are a few odd things that happen on the windows console perhaps we can chat about them elsewhere

Reply:
That's just because Windows is writing stderr to the same place that stdout and stdin are displayed; happens on Linux too. Pretty normal. (In a production setting, the output would be redirected to a log, which Caddy 2 can't do quite yet.)